app/screens/user_profile/user_info.tsx

```diff
@@ -17,36 +19,63 @@ type Props = {
     showNickname: boolean;
     showPosition: boolean;
     user: UserModel;
+    enableCustomAttributes?: boolean;
 }
 
-const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user}: Props) => {
-    const {formatMessage} = useIntl();
+const emptyList = [] as DisplayCustomAttribute[]; /** avoid re-renders **/
```

Comment:
```suggestion
const emptyList: DisplayCustomAttribute[] = []; /** avoid re-renders **/
```

Not a huge difference, but I don't like casting in typescript :P